packages/phishing-controller/src/PhishingController.ts

```diff
@@ -558,13 +606,29 @@ export class PhishingController extends BaseController<
       return;
     }
     const hotlist = hotlistResponse.data;
-    const newPhishingLists = this.state.phishingLists.map((phishingList) =>
-      applyDiffs(
+    const recentlyAdded = requestBlocklistResponse?.recentlyAdded || [];
```

Comment:
For follow up, if the hotlist were to fail or respond with an empty list, then the `requestBlocklist` is going to go out of sync. For example:
`// t=0 add{"abc.com"} remove{"123.com"} curr={"123.com"}` but hotlist failed or returned an empty list hence `curr` is not affected.
`// t=1 add{"def.com"} remove{} curr={"123.com", "def.com"}` which is incorrect but since the timestamp is shared and always updated regardless of success or failure, `123.com` will remain in the list until the stale call.

This problem also is preexisting for the `hotlist` as well.

Solution:
- Introduce separate `lastUpdated` variables for each list.
- Only update the `lastUpdated` if the calls are successful

Reply:
for now I think i will follow the same pattern following this comment: 
      // Set `stalelistLastFetched` and `hotlistLastFetched` even for failed requests to prevent server
      // from being overwhelmed with traffic after a network disruption.
      
I will leave this fix for a PR where i go in and do my a bigger refactor when openCTI is setup